code/__DEFINES/supermatter.dm

```diff
@@ -102,3 +92,64 @@
 #define SUPERMATTER_EMERGENCY 5
 /// Currently counting down to delamination. True [/obj/machinery/power/supermatter_crystal/var/final_countdown]
 #define SUPERMATTER_DELAMINATING 6
+
+// SUPERMATTER FACTORS DEFINES
+// This might feel obvious, but mentioning it doesnt hurt anyone:
+// While we separate these factors into neat little boxes, order still matters in the code.
+// There are a few factors that depends on other factors. So pay close attention on which values are interlinked.
+
+/// How much energy we get from external factors that are applied immediately.
+#define SM_POWER_EXTERNAL_IMMEDIATE "External Power Gain"
+/// How much energy we get from external factors that are applied over time.
+#define SM_POWER_EXTERNAL_TRICKLE "External Power Trickle"
+/// How much energy is gained from the temperature. Enabled by gas.
+#define SM_POWER_HEAT "Gas Heat Power Gain"
+/// How much energy the SM loses. Happens over time.
+/// Order matters here. We depend on current power + power gained from the factors above for the loss calc.
+#define SM_POWER_POWERLOSS "Internal Power Decay"
+/// How much of the energy the SM loses is recouped. From gas factors here.
+/// Order matters here. We depend on the powerloss amount.
+#define SM_POWER_POWERLOSS_GAS "Gas Power Decay Negation"
+/// How much of the energy the SM loses is recouped. From the psychologist this time.
+/// Order matters here. We depend on the powerloss amount.
+#define SM_POWER_POWERLOSS_SOOTHED "Psychologist Power Decay Negation"
+
```

Comment:
What... are these?

Reply:
The text we display in the ui breakdown. In power's case how much power is changing each tick.

Reply:
do we use them for tracking computation related values? or JUST for the ui

Reply:
I used them to index the assoc that we use for each `calculate_x()` proc, so the former.